opencti-platform/opencti-graphql/src/modules/indicator/indicator-converter.ts

```diff
@@ -27,7 +28,8 @@ const convertIndicatorToStix = (instance: StoreEntityIndicator): StixIndicator =
         ...indicator.extensions[STIX_EXT_OCTI],
         detection: instance.x_opencti_detection,
         score: instance.x_opencti_score,
-        main_observable_type: instance.x_opencti_main_observable_type
+        main_observable_type: instance.x_opencti_main_observable_type,
+        observables_values: getObservableValuesFromPattern(instance.pattern),
```

Comment:
```suggestion
        observables_values: getObservablesValuesFromPattern(instance.pattern),
```

Reply:
"s" of "observable values" removed everywhere